src/CMakeLists.txt

```diff
@@ -21,12 +26,22 @@ pkg_check_modules(SDL2 sdl2>=2.0.5 REQUIRED)
 pkg_check_modules(SDL2_TTF SDL2_ttf REQUIRED)
 pkg_check_modules(ZLIB zlib REQUIRED)
 pkg_check_modules(PNG libpng>=1.5 REQUIRED)
-pkg_check_modules(ZIP libzip>=1.0 REQUIRED)
+pkg_check_modules(ZIP libzip>=1.0)
+
+if(NOT ZIP_FOUND)
+	message(STATUS "  No suitable libzip found, data packaging will not be used.")
+	set(DISABLE_ZIP ON)
```

Comment:
Same thing here. If `libzip` is not found or too old, ask the user to install/upgrade it and exit with an error.

Reply:
Well, I agree that old cmake should not disable zip support at all, I'll fix it.

About the error: I've made this all while keeping in mind configure script of SDL2_mixer, which automatically disables support for file formats whose libraries do not exist in system (smpeg, ogg+vorbis+vorbisfile, modplug, etc.). But yes, there's opposite approach also exist: Mesa, for example, requires '--enable-gallium-llvm' manually specified in command line, even if there is '--gallium-drivers=radeonsi' exist, which is supposed to use LLVM; so the configure script fails with error, if I specify '--gallium-drivers=radeonsi' without '--enable-gallium-llvm', but adding the latter parameter to configure's command line makes configuration successful.

But is it really required to fail with an error if no suitable libzip found or we can't package files? Maybe warning should be enough?
I think about text like "Your version of CMake is too old, so packaging of data will not be used. Upgrade CMake to at least version 3.3 to enable packaging or specify PACKAGE_DATA=OFF to avoid this warning" and "You have no suitable libzip, so packaging of data will not be used and ZIP support will be disabled. Install at least libzip 1.0 to enable ZIP support or specify DISABLE_ZIP=ON to avoid this warning". Will this be ok?